eqcorrscan/core/match_filter/party.py

```diff
@@ -559,7 +561,16 @@ def decluster(self, trig_int, timing='detect', metric='avg_cor',
         """
         if self.__len__() == 0:
             return self
+
+        if min_chans > 0:
+            self.min_chans(min_chans - 1)
+            # min_chans uses explicit >, this uses >=
+
         all_detections = [d for fam in self.families for d in fam.detections]
+
+        if len(all_detections) == 0:
+            return Party()
+
```

Comment:
TODO: Check what these lines are about - they don't seem related to the topic of this PR?